### DISCUSSION TITLE
Relationship many-to-many not work

### DISCUSSION THREAD

**Discussion by jrujano** [aminalaee/sqladmin#740] [General]
I have a much to much relationship using 

https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#setting-bi-directional-many-to-many

 When I try to create the admin for this table, sqladmin tells me that it is not a table

![Captura de pantalla 2024-04-08 162249](https://github.com/aminalaee/sqladmin/assets/113458812/52d78a4e-76d9-4bb2-b504-b1623174399d)

**Comment by lukeclimen** [comment]
I think you want to make your `ServicioPrestatdorOrdenAdmin` using the python class associated with the table, not on the table itself. Take a look at this stack overflow post and see if it helps: https://stackoverflow.com/questions/18963567/in-sqlalchemy-whats-different-in-defining-table-using-base-and-table

**Comment by jrujano** [comment]
Taking into account the documentation
https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#many-to-many

The relationship is managed under an association table

Therefore, an attempt is made to perform said Admin through the table, but as we see it does not work.

To make this work I had to create the declaration of Said model, although in my opinion it is not the most optimal option, it worked worse.


![_admin](https://github.com/aminalaee/sqladmin/assets/113458812/defe65ef-73b8-4a23-94c4-88a505d84d89)
